tests/src/io.jsonwebtoken/jjwt-gson/0.11.5/src/test/java/io_jsonwebtoken/jjwt_gson/Jjwt_gsonTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package io_jsonwebtoken.jjwt_gson;
+
+import io.jsonwebtoken.*;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.io.Encoders;
+import io.jsonwebtoken.security.Keys;
+import org.junit.jupiter.api.Test;
+
+import javax.crypto.SecretKey;
+import java.util.Date;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class Jjwt_gsonTest {
+    @Test
+    void testSignedJWTs() {
+        SecretKey firstKey = Keys.secretKeyFor(SignatureAlgorithm.HS256);
```

Comment:
These and the other 2 tests seem to be identical, with the only change being which JSON library dependency we're importing in `build.gradle`. Would it be possible to somehow share these sources between the 3 tests? I'm not sure how exactly to do it though :/ cc @melix @dnestoro